### DISCUSSION TITLE
Wave Interval Plot Alignment

### DISCUSSION THREAD

**Discussion by shamilh2o** [h2oai/wave#1465] [Q&A] [answered]
I am using wave SDK 0.21.1 and plotted an interval plot. It gave an output as below.
<img width="1597" alt="image" src="https://user-images.githubusercontent.com/101506482/171405062-7865f9dc-6dc8-4c3a-a85e-3ef020628fba.png">

How can I adjust the edges of the X axis not to be in the corner of axis?

I saw that same thing happened in a plot in 'H2O Wave Demo' application as below.
<img width="338" alt="image" src="https://user-images.githubusercontent.com/101506482/171405945-4c319fcc-e51c-4668-82dc-e3d79d3c753f.png">

I am expecting an output somewhat similar to below. [https://wave.h2o.ai/docs/examples/plot-interval-stacked](url)
<img width="601" alt="image" src="https://user-images.githubusercontent.com/101506482/171405526-7209bf42-44ca-4980-a0f1-c2003eeca81c.png">
 
Can someone please guide me on how to fix it? 
Thanks in advance.

**Comment by mtanco** [comment, marked as answer]
You can change the type of your x-axis. I think you may be using a time or numeric type which will go to the ends of the plots (which is usually what we want for numeric data, but maybe not numeric internal plots). Instead try using the categorical type by adding the `x_scale="cat",` parameter to your `ui.mark`.

If you are still having problems please minimal example code, thanks!

**Reply by shamilh2o** [reply]
Thanks @mtanco for the clarification. It solved my issue.
